DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.13.03.SqlDataProvider

```diff
@@ -7,3 +7,38 @@
 /*****       for {databaseOwner} and {objectQualifier}  *****/
 /*****                                                  *****/
 /************************************************************/
+
+/* update folder permissions view to include default value for portalID GHI:8240*/

```

Comment:
Can we not put GHI:8240 which means nothing for us and the wider community please. We could put that on the issue/PR instead of in tracked files IMO.

Reply:
Fixed